public/js/scripts.js

```diff
@@ -362,14 +363,29 @@ function initializeCommentLayout(id) {
 function resizePosts() {
     for(var i = 0; i < getQuestionCount(); i ++) {
         var question = getQuestion(i);
+        
+        if(defaultPostSize == 0) {
+            defaultPostSize = parseInt($("#postContainer_" + question.getId()).css('height'));
+        }
+        
         var str = $("#textContainer_" + question.getId()).css('height');
         var num = parseInt(str);
```

Comment:
"str" and "num" were not very specific variable names. Took me a while to understand what they did